simulator/config/config.go

```diff
@@ -137,22 +138,26 @@ func getPort() (int, error) {
 
 // getKubeAPIServerURL gets KubeAPIServerURL from environment variable first, if empty from the config file.
 func getKubeAPIServerURL() string {
+	url := os.Getenv("KUBE_APISERVER_URL")
+	if url == "" && configYaml.KubeAPIServerURL != "" {
+		return configYaml.KubeAPIServerURL
+	}
 	p := os.Getenv("KUBE_API_PORT")
```

Comment:
Let's remove L145-160, but without removing KubeAPIPort and KubeAPIHost from `simulator/config/v1alpha1/types.go`. We can just ignore deprecated envs (as I said before, removing them would be a breaking change that we should avoid though